### ISSUE TITLE
Double-List

### ISSUE THREAD

**Issue by IzzySoft** [SmartPack/BusyBox-Installer#10]
Forgot to ask you, @sunilpaulmathew – shall I once more keep the app in my repo, now that it's available at F-Droid? Or would that be "overkill"?

**Comment by sunilpaulmathew** [comment]
> Forgot to ask you, @sunilpaulmathew – shall I once more keep the app in my repo, now that it's available at F-Droid? Or would that be "overkill"?

No problem. However, I have to restart providing APK's with releases (stopped with last release for the sake of F-Droid). Also, I don't realised you already included app in your repo. Otherwise, I couldn't stopped it. Will restart from next release (not at all a problem for me).

**Comment by IzzySoft** [comment]
Well, we have both options – whichever you prefer. You know, usually I remove apps from my repo once they become available at F-Droid, so removal would be the default. I just remembered you wanted me to keep the last one already, so I asked.

And yeah… sometimes I pick apps when an RFP is opened, as until the app becomes available at F-Droid might take a while.

**TL;DR:** just tell me your preference, and I'll accept :wink:

**Comment by sunilpaulmathew** [comment]
> Well, we have both options – whichever you prefer. You know, usually I remove apps from my repo once they become available at F-Droid, so removal would be the default. I just remembered you wanted me to keep the last one already, so I asked.
> 
> And yeah… sometimes I pick apps when an RFP is opened, as until the app becomes available at F-Droid might take a while.
> 
> **TL;DR:** just tell me your preference, and I'll accept 😉

Then, let's keep it in both repos? I'll gladly provide APKs aith next release :)

**Comment by IzzySoft** [comment]
[Done](https://apt.izzysoft.de/fdroid/index/apk/com.smartpack.busyboxinstaller) (see the note at the end of the description) – thanks!
